README.md

```diff
@@ -105,6 +105,8 @@ Ruby gem: https://github.com/ankane/lightgbm
 
 LightGBM4j (Java high-level binding): https://github.com/metarank/lightgbm4j
 
+lightgbm_ray (LightGBM on Ray): https://github.com/ray-project/lightgbm_ray
```

Comment:
I suggest moving Ray after `Kubeflow Operator` line so that all distributed frameworks will be listed closely to each other. In this list we try to follow logical grouping not alphabetical one.

Reply:
oh right, ok. Moved in https://github.com/microsoft/LightGBM/pull/4584/commits/75916754f5fc634acb84fc7ad3873b90cc68a0f5